frontend/src/views/GNewShootEditor.vue

```diff
@@ -134,7 +135,16 @@ onBeforeRouteLeave(async (to, from, next) => {
   if (isShootCreated.value) {
     return next()
   }
-  if (isShootDirty.value) {
+  if (!clean.value) {
+    if (!await confirmEditorNavigation()) {
+      focusEditor()
+      return next(false)
+    }
+  }
+  return next()
+})
+onBeforeRouteUpdate(async (to, from, next) => {
```

Comment:
What is the reason for this change. I think this has nothing to do with cm6. Isnt't this part of an ohter PR?

Reply:
reverted